Warning \"moving a temporary object prevents copy elision\"

Warning raised: 

```shell
src/multipatch/data_types/multipatch_field.hpp:91:76: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
 91 | : base_type(std::move(T<Patches>(other.template get<Patches>()))...)
 | ^
src/multipatch/data_types/multipatch_field.hpp:91:76: note: remove ‘std::move’ call
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -21,7 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix break points incorrectly labelled as knots.
 - Fix minimum version requirement of Kokkos.
 - Fix tolerance of floating point comparisons in JacobianMatrixAndJacobianCoefficients and MultipatchSplineEvaluatorTest tests
-- Fix unnecessary std::move calls
+- Fix unnecessary `std::move` calls.
 
 ### Changed
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix break points incorrectly labelled as knots.
 - Fix minimum version requirement of Kokkos.
 - Fix tolerance of floating point comparisons in JacobianMatrixAndJacobianCoefficients and MultipatchSplineEvaluatorTest tests
+- Fix unnecessary std::move calls
 
 ### Changed
 

--- a/src/matrix_tools/matrix_batch_csr.hpp
+++ b/src/matrix_tools/matrix_batch_csr.hpp
@@ -137,16 +137,11 @@ public:
                 batch_sparse_type::
                         create(gko_exec,
                                gko::batch_dim<2>(batch_size(), gko::dim<2>(size(), size())),
-                               std::move(gko::array<double>::
-                                                 view(gko_exec,
-                                                      batch_values.span(),
-                                                      batch_values.data())),
-                               std::move(gko::array<
-                                         int>::view(gko_exec, cols_idx.span(), cols_idx.data())),
-                               std::move(gko::array<int>::
-                                                 view(gko_exec,
-                                                      nnz_per_row.span(),
-                                                      nnz_per_row.data()))));
+                               gko::array<double>::
+                                       view(gko_exec, batch_values.span(), batch_values.data()),
+                               gko::array<int>::view(gko_exec, cols_idx.span(), cols_idx.data()),
+                               gko::array<int>::
+                                       view(gko_exec, nnz_per_row.span(), nnz_per_row.data())));
     }
 
     /**

--- a/src/multipatch/data_types/multipatch_field.hpp
+++ b/src/multipatch/data_types/multipatch_field.hpp
@@ -100,7 +100,7 @@ public:
      */
     template <class MultipatchObj, std::enable_if_t<!is_mem_type_v<MultipatchObj>, bool> = true>
     KOKKOS_FUNCTION MultipatchField(MultipatchObj& other)
-        : base_type(std::move(T<Patches>(other.template get<Patches>()))...)
+        : base_type(T<Patches>(other.template get<Patches>())...)
     {
         static_assert(is_multipatch_type_v<MultipatchObj>);
     }
@@ -116,7 +116,7 @@ public:
      */
     template <class MultipatchObj, std::enable_if_t<is_mem_type_v<MultipatchObj>, bool> = true>
     explicit MultipatchField(MultipatchObj& other)
-        : base_type(std::move(T<Patches>(other.template get<Patches>()))...)
+        : base_type(T<Patches>(other.template get<Patches>())...)
     {
         static_assert(is_multipatch_type_v<MultipatchObj>);
     }

--- a/src/multipatch/data_types/multipatch_field_mem.hpp
+++ b/src/multipatch/data_types/multipatch_field_mem.hpp
@@ -91,7 +91,7 @@ public:
      */
     template <class MultipatchObj>
     explicit MultipatchFieldMem(MultipatchObj& other)
-        : base_type(std::move(T<Patches>(other.template get<Patches>()))...)
+        : base_type(T<Patches>(other.template get<Patches>())...)
     {
         static_assert(is_multipatch_type_v<MultipatchObj>);
     }

